x-pack/plugins/uptime/public/apps/plugin.ts

```diff
@@ -79,11 +79,11 @@ export class UptimePlugin
 
     if (plugins.observability) {
       plugins.observability.dashboard.register({
-        appName: 'uptime',
+        appName: 'synthetics',
```

Comment:
Why are we changing the app name?

Reply:
i think it's a better name, since this is actually not an app name, it's data handler name, instead of uptime data handler, synthetics data hanlder make more sense and also it makes life easier in terms of types, since i can use the same types in exploratory view.